test/integration/bad/stanc.expected

```diff
@@ -1379,6 +1379,28 @@ Semantic error in 'lp-error.stan', line 5, column 2 to column 6:
    -------------------------------------------------
 
 Identifier 'lp__' clashes with reserved keyword.
+  $ ../../../../install/default/bin/stanc lvalue_indexes1.stan
```

Comment:
Can you also add tests showing the warning?

Reply:
Sure, where do they go? `test/integration/bad/` is for models that don't compile, right? `test/integration/good/` has some models with deprecation warnings so is that the right place? `test/integration/good/lvalue_multiindex.stan`

Reply:
I think we may want to start thinking about #1049 and make a new folder `test/integration/good/deprecated` or something, with this there. We can then slowly start moving things there and eventually run the canonicalizer on anything we don't want to end up in said folder

Reply:
After thinking about it, `test/integration/good/warning` or `test/integration/good/emits_warning` may be best (we can also put things like empty.stan there)